ui/app/templates/workspace/projects/project/app/deployment/deployment-seq.hbs

```diff
@@ -8,7 +8,7 @@
             href={{enforce-protocol @model.preload.deployUrl}}
             class="button button--secondary button--external-link">
             <span>{{lowercase @model.preload.deployUrl}}</span>
-            <Pds::Icon @type="exit" class="icon" />
+            <FlightIcon @name="exit" class="icon" />
```

Comment:
Maybe `exit` isn’t the icon name?

<img width="381" alt="CleanShot 2021-12-09 at 10 05 58@2x" src="https://user-images.githubusercontent.com/34030/145376079-8bec505f-e505-4d50-b0cc-94ac782ee2fa.png">